app/views/sign_up_sheet/review_bids_others_work.html.erb

```diff
@@ -1,113 +1,103 @@
-<% session[:return_to] = request.url %>
-<% if flash[:notice] != nil and flash[:notice].strip != "" %>
-       <div class="flash_error">
-       <%= flash[:notice] %>
-       </div>
+<% def flash_message %>
+  <% session[:return_to] = request.url %>
+  <% if flash[:notice] != nil and flash[:notice].strip != "" %>
+    <div class="flash_error">
+    <%= flash[:notice] %>
+    </div>
+  <% end %>
+<% end %> 
+
+
+<% def display_topic_and_participant(title, review_no, topic_id, assignment, participant) %> 
+  <td>
+  <b><%= "#{title} #{review_no}." %></b>
+  # --In ‘review_bids/index’ page, if the topic_identifier is empty, ‘:’ sign should not be displayed.-->
+  <% topic = SignUpTopic.find(topic_id) %>
+  <% if !topic_id.nil? %>
+    <%= "#{topic.topic_identifier.present? ? "#{topic.topic_identifier}: " : ""}#{topic.topic_name}" %>
+  <%end%>
+  <%if !@assignment.is_anonymous%>
+    &nbsp; (author: <%= participant.fullname%>)
+  <%end%>
+  </td>
 <% end %>
 
-<h2>Reviews for "<%= @assignment.name %>"</h2>
 
-<!-- add headers that provide information regarding review numbers -->
-<% if @assignment.num_reviews_allowed.nil? || @assignment.num_reviews_allowed == -1%>
-    <h4>Your instructor expects you to do <%= @assignment.num_reviews_required %> reviews. You are not allowed to do any extra reviews. </h4>
-<% elsif @assignment.num_reviews_allowed == @assignment.num_reviews_required %>
-    <h4>You should perform exactly <%= @assignment.num_reviews_allowed %> reviews </h4>
-<% else %>
-    <h4>You may perform between <%= @assignment.num_reviews_required %> and <%= @assignment.num_reviews_allowed %> reviews</h4>
+<% def display_assignment_reviews_info(assignment, num_reviews_completed) %>
+  <h2>Reviews for "<%= assignment.name %>"</h2>
+  <!-- add headers that provide information regarding review numbers -->
+  <% if assignment.num_reviews_allowed.nil? || assignment.num_reviews_allowed == -1 %>
```

Comment:
Why would the number of reviews allowed ever be negative?  If -1 represents a flag, there are better ways to express an exceptional condition.